Implicit "any" issues

<!-- Issue Template -->

#### Environment(required)

- antd version: 2.8.0
- OS and its version: Ubuntu 16.04

```
 ERROR in antd/lib/table/SelectionCheckboxAll.d.ts
 (13,30): error TS7006: Parameter 'item' implicitly has an 'any' type.
 
 ERROR in antd/lib/table/SelectionCheckboxAll.d.ts
 (13,36): error TS7006: Parameter 'index' implicitly has an 'any' type.
 
 ERROR in antd/lib/table/SelectionCheckboxAll.d.ts
 (14,20): error TS7006: Parameter 'record' implicitly has an 'any' type.
 
 ERROR in antd/lib/table/SelectionCheckboxAll.d.ts
 (14,28): error TS7006: Parameter 'index' implicitly has an 'any' type.
 ```

--- a/components/table/SelectionCheckboxAll.tsx
+++ b/components/table/SelectionCheckboxAll.tsx
@@ -15,8 +15,8 @@ export interface SelectionCheckboxAllProps {
   store: Store;
   locale: any;
   disabled: boolean;
-  getCheckboxPropsByItem: (item, index) => any;
-  getRecordKey: (record, index?) => string;
+  getCheckboxPropsByItem: (item: any, index: number) => any;
+  getRecordKey: (record: any, index?: number) => string;
   data: any[];
   prefixCls: string | undefined;
   onSelect: (key: string, index: number, selectFunc: any) => void;

